<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProfileDisplayDialog</class>
 <widget class="QDialog" name="ProfileDisplayDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>个人信息</string>
  </property>
  <property name="fixedSize" stdset="0">
   <size>
    <width>400</width>
    <height>500</height>
   </size>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>15</number>
   </property>
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <widget class="QLabel" name="lblAvatar">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>120</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>120</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #f0f0f0; border-radius: 60px; border: 1px solid #eee;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="accountLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="lblAccountLabel">
       <property name="styleSheet">
        <string notr="true">font-size: 14px; color: #333; min-width: 60px; text-align: right;</string>
       </property>
       <property name="text">
        <string>账号：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblUsername"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="nicknameLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="lblNicknameLabel">
       <property name="styleSheet">
        <string notr="true">font-size: 14px; color: #333; min-width: 60px; text-align: right;</string>
       </property>
       <property name="text">
        <string>昵称：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblNickname"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="genderLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="lblGenderLabel">
       <property name="styleSheet">
        <string notr="true">font-size: 14px; color: #333; min-width: 60px; text-align: right;</string>
       </property>
       <property name="text">
        <string>性别：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblGender"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="phoneLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="lblPhoneLabel">
       <property name="styleSheet">
        <string notr="true">font-size: 14px; color: #333; min-width: 60px; text-align: right;</string>
       </property>
       <property name="text">
        <string>手机号：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblPhone"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="emailLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="lblEmailLabel">
       <property name="styleSheet">
        <string notr="true">font-size: 14px; color: #333; min-width: 60px; text-align: right;</string>
       </property>
       <property name="text">
        <string>邮箱：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblEmail"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="btnEditProfile">
     <property name="text">
      <string>编辑资料</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
